<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_acc</class>
 <widget class="QWidget" name="add_acc">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Mongolian Baiti</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>add_acc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#add_acc{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(66, 134, 201, 255));
border-radius:10px;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1001</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>35</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>38</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>MJDB</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00/00/00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Utilisateur:</string>
    </property>
   </widget>
   <widget class="QLabel" name="user">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="title_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>34</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>34</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrale{
	border-radius:15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_5">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>44</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>17</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="QPushButton" name="mini">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	border-radius:10px;
	background-color: rgb(255, 170, 0);
}

QPushButton:hover{
	background-color: rgb(206, 137, 0);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_btn">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	border-radius:10px;
	background-color: rgb(255, 0, 0);
}

QPushButton:hover{
	background-color: rgb(211, 0, 0);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GESTION DES COMPTES</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>90</y>
      <width>1011</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Compte:</string>
    </property>
   </widget>
   <widget class="QLabel" name="selcted_acc">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stack">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>99</y>
     <width>1001</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>300</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit">
     <property name="geometry">
      <rect>
       <x>792</x>
       <y>300</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Editer</string>
     </property>
    </widget>
    <widget class="QLabel" name="wrong">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>239</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_sc">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>340</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nouveau sous-compte</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>894</x>
       <y>300</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nom du compte:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;
border-bottom:1px solid #9f9f9f;
</string>
     </property>
     <property name="placeholderText">
      <string>Nom du compte</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Déscription:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="desc">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>100</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: rgb(0, 0, 0);
border-bottom:1px solid rgb(0, 0, 0);
border-right:1px solid rgb(0, 0, 0);
border-radius:15px;
</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Mongolian Baiti'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Facultatif</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="list_acc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>621</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>34</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>199</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>8</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <column>
      <property name="text">
       <string>Date/Haure</string>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>200</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Remarque obligatoire</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="list_acc_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>621</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>34</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>149</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>8</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <column>
      <property name="text">
       <string>Compte</string>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date/Haure</string>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QTextEdit" name="desc_2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>100</y>
       <width>231</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: rgb(0, 0, 0);
border-bottom:1px solid rgb(0, 0, 0);
border-right:1px solid rgb(0, 0, 0);
border-radius:15px;
</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Mongolian Baiti'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Facultatif</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_3">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>360</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>40</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;
border-bottom:1px solid #9f9f9f;
</string>
     </property>
     <property name="placeholderText">
      <string>Nom du compte</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Déscription:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>360</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Editer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nom du sous-compte:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>360</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:10px;
	color: rgb(255, 255, 255);
	
	background-color:none;
}
QPushButton:hover {
	
	background-color: rgb(0, 230, 169);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/back_arrow-01.png</normaloff>icons/back_arrow-01.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>34</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="wrong_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>270</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
